fuseimg/utils/visualization/imaging_multi_plot.py

```diff
@@ -0,0 +1,99 @@
+from typing import List , Callable , Any
+import matplotlib.pyplot as plt
+import numpy as np
+from functools import partial
+from fuseimg.utils.typing.typed_element import TypedElement
+from matplotlib.collections import PatchCollection
+from matplotlib.patches import Polygon
+
+def show_multiple_images(plot_seg : Callable , imgs : List, base_resolution :int = 5, **args ):
+    '''
+    Show multiple images with shared zoom/translation controls
+    everything passed as kwargs (for example - cmap='gray') will be passed to the individual imshow calls
+    special possible values are:
+        * cmap[image_index] = value
+            for example - cmap0='gray' will only change the cmap of the first image
+        * unify_size = 'dont_care'
+            will make sure that all images are resized to the same size
+    example usage:
+    imshowmultiple([img1, img2])
```

Comment:
show_multiple_images
Also the first argument is plot_seg

Reply:
updated